packages/graphql/tests/utils/tests-helper.ts

```diff
@@ -92,15 +91,21 @@ export class TestHelper {
         return this.neo4jGraphQL;
     }
 
-    public async isCDCEnabled(): Promise<boolean> {
+    public async assertCDCEnabled(): Promise<boolean> {
```

Comment:
Not important for now: But I would not run an `ALTER DATABASE` inside a method called `isCDCEnabled` 

edit: I notice now is changed to `assertCDCEnabled` however this method could return false for Aura instances.

Not a blocker for the release IMO anyway!

Reply:
we need to improve the ergonomics, I called assert for the reason uou mention, I just want to unblock release atm, later we can chat on improving this